boards/st/stm32f411e_disco/stm32f411e_disco_stm32f411xe_B.overlay

```diff
@@ -29,3 +30,12 @@
 			  <&gpioe 5 GPIO_ACTIVE_HIGH>;
 	};
 };
+
+&spi1 {
+	/delete-node/ i3g4250d@0;
+	/*l3gd20: l3gd20@0 {
+		compatible = "st,l3gd20";
+		reg = <0>;
+		spi-max-frequency = <1000000>;
+	};*/
```

Comment:
Is this left commented on purpose ?

Reply:
Yes, I left it in but I don't have a board to test it on.  I will remove it for now.